go.mod

```diff
@@ -4,15 +4,15 @@ go 1.20
 
 require (
 	github.com/Azure/azure-storage-blob-go v0.7.0
-	github.com/VictoriaMetrics/fastcache v1.6.0
```

Comment:
need change go.mod `go directive` 

```
go 1.20
```

Reply:
updated in [3640e89](https://github.com/scroll-tech/go-ethereum/pull/726/commits/3640e8938d2f03d23201fe9cf18bfe3adc41782f).